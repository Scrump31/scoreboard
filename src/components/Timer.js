import React from 'react';

export default function Timer() {
  return (
    <div>
      00:00
    </div>
  );
}
