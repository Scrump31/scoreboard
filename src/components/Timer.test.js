import React from 'react';
import { shallow } from 'enzyme';
import Timer from './Timer';

describe('<Timer />', () => {
  it('renders without crashing', () => {
    const component = shallow(<Timer />);
    expect(component).toMatchSnapshot();
  });
});
