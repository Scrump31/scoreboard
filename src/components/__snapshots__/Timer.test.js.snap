// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Timer /> renders correctly 1`] = `
<div>
  <div
    className="time"
  >
    1:00
  </div>
  <div
    className="progress"
  >
    <span
      style={
        Object {
          "width": "0%",
        }
      }
    />
  </div>
  <div
    className="controls"
  >
    <button
      className="start btn btn-sm btn-success"
      onClick={[Function]}
    >
      Start
    </button>
    <button
      className="pause btn btn-sm btn-default"
      onClick={[Function]}
    >
      Pause
    </button>
    <button
      className="stop btn btn-sm btn-danger"
      onClick={[Function]}
    >
      Stop
    </button>
  </div>
</div>
`;
